### ISSUE TITLE
SMAR-482 ⁃ Simple routing command does not work

### ISSUE THREAD

**Issue by thatdoogieguy** [thatdoogieguy/SST#522]
When going into routing the platform still does not allow you to select a start and end point

Error:

Error
Trip is not possible. Your start or end point might not be safely accessible (for instance, you might be starting on a residential street connected only to a highway).

!image-2018-01-29-09-17-30-462.png|thumbnail!

┆Attachments: <a href="https://smartsensor.atlassian.net/secure/attachment/24779/image-2018-01-29-09-17-30-462.png">image-2018-01-29-09-17-30-462.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/24784/screenshot-1.png">screenshot-1.png</a>

**Comment by thatdoogieguy** [comment]
➤ User  commented:

We discussed this very issue the other day. Elements that are not street accessible are not easily routable.

We also discussed routing in depth the other day and also addressed the priority of it in the Q1 plan.

Why have a plan if we are not going to stick to it?

**Comment by thatdoogieguy** [comment]
➤ User  commented:

Hi Ian,

I understand we have a plan and a sprint and a road map.

What I was asking the system to do was not something that hasnt been done before. 

I selected Esplanade Zone, which is only on a road - The Esplanade on the Gold Coast. 

I selected start at the top of the Esplanade and end at the bottom of the Esplanade.

This was a simple command show I could show in my report. 

It was not that I expected to disrupt the "Plan" but to use the basic commands that have been developed to date.

Regards,

Leon

!screenshot-1.png|thumbnail!
